src/pages/present/team.tsx

```diff
@@ -1,20 +1,158 @@
-import { HeadProps } from 'gatsby';
+import { css, Theme, useTheme } from '@emotion/react';
+import { graphql, HeadProps, PageProps } from 'gatsby';
+import { GatsbyImage, getImage } from 'gatsby-plugin-image';
+import { Markup } from 'interweave';
 import { FormattedMessage } from 'react-intl';
+import { DividedSection } from '../../components/DividedSection';
+import { SummaryParagraph } from '../../components/SummaryParagraph';
 import { Document } from '../../layouts/default/Document';
+import { ContentMain } from '../../layouts/default/types/ContentMain';
 import { DefaultLayout } from '../../layouts/DefaultLayout';
 
-export default function TeamPage() {
+function TeamHeader({ summary }: { summary?: string | null }) {
+  const theme = useTheme();
   return (
-    <DefaultLayout>
-      <section>
-        <h1>
-          <FormattedMessage id="page.present.team.meta.title" />
-        </h1>
-      </section>
+    <DividedSection upperColor={theme.backgroundColor} lowerColor={theme.whiteColor}>
+      <h1>
+        <FormattedMessage id="page.present.team.meta.title" />
+      </h1>
+      <SummaryParagraph>
+        <Markup content={summary} />
+      </SummaryParagraph>
+    </DividedSection>
+  );
+}
+
+const portraitsSectionStyle = (theme: Theme) => css`
+  background-color: ${theme.whiteColor};
+  flex-grow: 1;
+
+  #team-portraits {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr 1fr;
+    gap: 3rem;
+    max-width: 100%;
+
+    @media (max-width: ${theme.breakpoints.small}) {
+      grid-template-columns: 1fr 1fr;
+    }
+
+    .portrait {
+      display: flex;
+      flex-direction: column;
+
+      img {
+        width: 100%;
+      }
+
+      p {
+        margin-top: 0.5rem;
+        text-align: center;
+      }
+    }
+  }
+`;
+
+function PortraitsSection({ portraits }: { portraits: Queries.TeamPageQuery['portraits']['nodes'] }) {
+  const theme = useTheme();
+  return (
+    <DividedSection upperColor={theme.whiteColor} lowerColor={theme.backgroundColor} flipVertically css={portraitsSectionStyle}>
+      <h2>
+        <FormattedMessage id="page.present.team.section.portraits" />
```

Comment:
Ich würde diese Überschrift sein lassen oder verstecken, wenn es SEO relevant ist.

Reply:
Genau sowas habe ich mir auch überlegt, jedoch als ich nachgeforscht habe, ob eine versteckte Überschrift in Ordnung für SEO ist, habe ich Resultate gefunden, dass dies dem SEO schadet, weil es ein Dark Pattern sein könnte, um SEO zu manipulieren. Gerne würde ich die Überschrift mal so stehen lassen.